packages/ng/core-select/input/select-input.component.ts

```diff
@@ -147,6 +147,7 @@ export abstract class ALuSelectInputComponent<TOption, TValue> implements OnDest
 
 	public valueSignal = signal<TValue>(null);
 	isFilterPillEmpty = computed(() => this.valueSignal() === null);
+	isFilterPillClearable = computed(() => this.clearable);
```

Comment:
Won't this always be false because of the computed being run before the component is initialized, thus returning false?